### DISCUSSION TITLE
Issue with Gazelle trackers to find releases

### DISCUSSION THREAD

**Discussion by GregTroar** [cross-seed/cross-seed#247] [General]
The search is made with torznab link with movie title + year (ie: https://xxxxx.com/api/v2.0/indexers/xxxxxx/results/torznab/api?apikey=[REDACTED]&t=search&q=Dolittle+2020)

Actually the fact that the year is used return no results while without the year the results are returned.
Is there anyway to solve this ?

**Comment by mmgoodnow** [comment]
Maybe, this would be an interesting thing for someone with a large collection to test.

**Reply by Undrog** [reply]
What did you have in mind to test?

I edited utils.js to use movieMatch.groups.title instead of movieMatch[0] - I am running this through a set of 427 torrent now and it looks to be working. I guess the consequences will be more non-matching results to process.

Maybe searching without year can be a per torznab URL option in the config file?

Did you have something more clever in mind?

**Comment by kzmgd** [comment]
Searching with year appended with a space is not a great practice. For example, how do you know to search for "Blade Runner 2049" vs a movie called "Blade Runner" that came out in 2049? (Maybe not a realistic scenario for 2 decades, but still...)

I'm a dev at Anthelion who recently implemented a torznab search. I discovered Prowlarr uses a non-standard formatting for including year in the query string that looks like:

`https://xxxxx.com/api/v2.0/indexers/xxxxxx/results/torznab/api?apikey=[REDACTED]&t=search&q=Dolittle{Year:2020}`

Might be worth investigating whether that works. Alternatively, at least for us, we've created a separate parameter to do this query in a more sane format:

`.../api?apikey=[REDACTED]&t=search&q=Dolittle&year=2020`

**Reply by bakerboy448** [reply]
What does the Torznab caps call report?

also if doing a manual search in prowlarr it doesn't support the year parameter if you're doing a basic search (default) would need to be a typed search

**Reply by Undrog** [reply]
> I'm a dev at Anthelion who recently implemented a torznab search. I discovered Prowlarr uses a non-standard formatting for including year in the query string that looks like:
> 
> `https://xxxxx.com/api/v2.0/indexers/xxxxxx/results/torznab/api?apikey=[REDACTED]&t=search&q=Dolittle{Year:2020}`

I have tried your example search in Prowlarr: "Dolittle{Year:2020}". ANT is the only tracker that I have that returns a result for this query, all other trackers that I have, which all have the release, find nothing for"Dolittle{Year:2020}", but they do all find the results for "Dolittle 2020".

I suspect you may have misunderstood how Prowlarr works, as I cannot find any intuitive way to build the query you mention in Prowlarr other than to type it manually. For example, I can click the button next to the search box, and can change it to a Movie Search, but then Prowlarr is **not** supporting search by title, they are only supporting search by imdbid, tmdbit, and year (which is a little strange). Any attempt to combine the "Year" with a normal title search finds no results on any tracker (except for with ANT, as you've specifically enabled this format to work)

**Reply by bakerboy448** [reply]
text query/title search is supported for basic and all search types

**Reply by Undrog** [reply]
> text query/title search is supported for basic and all search types

Could you provide an example of how this works with Prowlarr? I guess I am the one misunderstanding. My attempts so far to combine text query/title search with the Movie Search type fail on all trackers except for ANT for me.. Should the example provided "Dolittle{Year:2020}" work for a Movie Search?

**Reply by Qstick** [reply]
> I'm a dev at Anthelion who recently implemented a torznab search. I discovered Prowlarr uses a non-standard formatting for including > year in the query string that looks like:

> https://xxxxx.com/api/v2.0/indexers/xxxxxx/results/torznab/api?apikey=[REDACTED]&t=search&q=Dolittle{Year:2020}

This is incorrect, that format is only supported in Prowlarr UI, the torznab api is completely torznab compliant. Year is not a valid parameter for a `t=search` (a `t=caps` call will show you this). `&year=2000` can be used for any other search type based on the indexer capabilities.

**Reply by bakerboy448** [reply]
should be fixed for the UI as well now
https://github.com/Prowlarr/Prowlarr/commit/810b3612aadb0fc63c2de80af1517bd9df375c20

**Reply by kzmgd** [reply]
Excellent! I'm glad that was just a weird thing I must've seen once in Prowlarr testing, and I'll gladly remove that case in our code. Also good to hear that year is expected with movie searches (at least per caps, which ours does include, too).

**Reply by Undrog** [reply]
> should be fixed for the UI as well now [Prowlarr/Prowlarr@810b361](https://github.com/Prowlarr/Prowlarr/commit/810b3612aadb0fc63c2de80af1517bd9df375c20)

Awesome, looks to be working great! Thanks @bakerboy448!

**Comment by newadventure079** [comment]
@mmgoodnow I ran into this same issue also.  cross-seed searched for `&t=tvsearch&q=Another+Life+2019&season=1`  and it was not found on HDB.  HDB has it listed as `Another Life S01 2160p WEBRip DD+5.1 x264-NTb`
